src/main/resources/webroot/index.html

```diff
@@ -108,6 +108,15 @@ <h2>Create a new Vert.x application</h2>
             </div>
           </div>
         </div>
+        <div class="form-group row">
+          <label class="col-sm-4 col-form-label">Flavor</label>
+          <div class="col-sm-8">
+            <div class="btn-group">
+              <label class="btn btn-sm btn-default" ng-repeat="flavor in vm.flavors"
```

Comment:
We need some logic to disable dependencies that don't have Mutiny bindings when the Mutiny flavor is selected.

Reply:
Is there a practical reason why it shouldn't be allowed? What happens if a user wants to create a project with the mutiny flavor but wants to use the zipkin artifact too?

Reply:
I added a similar message to that of the vertx-unit / junit5 combination in dc01e0c. WDYT?